content/en/user/profile.md

```diff
@@ -79,8 +79,11 @@ If you put an HTTPS link in your profile metadata, Mastodon checks if that link
 
 More precisely, Mastodon will validate the link under the following conditions:
 - Since 4.0: the hostname does not change after IDN normalization
+- the link to your Mastodon profile is added to a static HTML page used in your site. Websites served via Javascript-heavy content management systems such as StoryBlok often do not include static HTML, but check with your site developer
 - it starts with HTTPS
 - the resolved page contains at least one `a` or `link` tag with a `rel="me"`
 - the `href` attribute on one of those elements is equal to the URL for your Mastodon profile
 
-Alternatively, validation will occur if the resolved page's *first* link has an `href` value that redirects to your Mastodon profile's URL (such as through a link shortener).
\ No newline at end of file
+Alternately, validation will occur if the resolved page's *first* link has an `href` value that redirects to your Mastodon profile's URL (such as through a link shortener).
+
+If validation succeeds, the link to your website will turn green on your profile page. This may take a few minutes even after you deploy the update to your site's HTML. 
```

Comment:
In fact, Mastodon won't "re-check" your page if it fails validation the first time. So you'll need to remove and then and re-add the link to your profile after making the change to your site's HTML before it will check again to see if it's been updated.

Reply:
This was not my experience last night, but it's a more orderly set of steps. Made the change, and removed the newline from the end of the file.